src/chapters/how-to-code/print-function.rst

```diff
@@ -64,5 +64,5 @@ combination ``\n``.
       print("Python\nJavaScript\nJava\nC#\nSwift")
 
 In addition to the newline character, there is also a special tab character,
-``\t``. Go back to the six examples and play around with using ``\t``
+``\t``. Go back to the above examples and play around with using ``\t``
```

Comment:
"above examples" vs. "examples above" - what's your preference?  Most of the time, I use the latter, but a quick Google search says either way is acceptable.

I'll leave the final choice to you.

Reply:
I wasn't actually sure what to put here! Just for clarification, this sentence was specifying that the students should refer back to the examples that are earlier on the page? If so, I like "the above examples".